lib/client/keystore.go

```diff
@@ -223,9 +223,27 @@ func (fs *FSLocalKeyStore) DeleteUserCerts(idx KeyIndex, opts ...CertOption) err
 
 // DeleteKeys removes all session keys.
 func (fs *FSLocalKeyStore) DeleteKeys() error {
-	if err := os.RemoveAll(fs.KeyDir); err != nil {
+	const dontDeleteTshConfigFileName = "config"
+	files, err := os.ReadDir(fs.KeyDir)
+	if err != nil {
 		return trace.ConvertSystemError(err)
 	}
+	for _, file := range files {
+		if file.IsDir() && file.Name() == dontDeleteTshConfigFileName {
```

Comment:
Can you add a test case that makes sure that tsh config file does not get removed on tsh logout?